test/jdk/javax/swing/JMenuItem/RightLeftOrientation.java

```diff
@@ -54,64 +72,70 @@
 import javax.swing.KeyStroke;
 import javax.swing.LookAndFeel;
 import javax.swing.SwingConstants;
+import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
 
 public class RightLeftOrientation {
 
+    static volatile JFrame frame;
+
     private static final String INSTRUCTIONS = """
-        A menu bar is shown containing a menu for each look and feel.
-        A disabled menu means that the look and feel is not available for
-        testing in this environment.
-        Every effort should be made to run this test
-        in an environment that covers all look and feels.
+        A menu bar is shown with a menu.
 
-        Each menu is divided into two halves. The upper half is oriented
+        The menu is divided into two halves. The upper half is oriented
         left-to-right and the lower half is oriented right-to-left.
-        For each menu, ensure that the lower half mirrors the upper half.
+        Ensure that the lower half mirrors the upper half.
 
         Note that when checking the positioning of the sub-menus, it
         helps to position the frame away from the screen edges.""";
 
     public static void main(String[] args) throws Exception {
+        if (args.length < 1) {
+            throw new IllegalArgumentException("Look-and-Feel keyword is required");
+        }
+        final String lafClassName;
+        switch (args[0]) {
+            case "metal" -> lafClassName = UIManager.getCrossPlatformLookAndFeelClassName();
+            case "motif" -> lafClassName = "com.sun.java.swing.plaf.motif.MotifLookAndFeel";
+            case "windows" -> lafClassName = "com.sun.java.swing.plaf.windows.WindowsLookAndFeel";
+            default -> throw new IllegalArgumentException(
+                           "Unsupported Look-and-Feel keyword for this test: " + args[0]);
+        }
+        SwingUtilities.invokeAndWait(() -> {
+            try {
+                UIManager.setLookAndFeel(lafClassName);
+            } catch (Exception e) {
+                throw new RuntimeException(e);
+            }
+        });
+        System.out.println("Test for LookAndFeel " + lafClassName);
         PassFailJFrame.builder()
                 .title("RightLeftOrientation Instructions")
                 .instructions(INSTRUCTIONS)
                 .columns(35)
                 .testUI(RightLeftOrientation::createTestUI)
                 .build()
                 .awaitAndCheck();
+       System.out.println("Test passed for LookAndFeel " + lafClassName);
```

Comment:
jtr file is available for each L&F testing and by default "Test Passed!" output is available. This doesn't seem to be useful.

     Test for LookAndFeel javax.swing.plaf.metal.MetalLookAndFeel
     Test passed!
     Test passed for LookAndFeel javax.swing.plaf.metal.MetalLookAndFeel

Reply:
ok